### PR TITLE
Add SceGxm nids

### PR DESCRIPTION
**Pull request by Princess-of-Sleeping**

### CODE REVIEW

File: db.yml [hunk 1]
```diff
@@ -2287,11 +2287,43 @@ modules:
...
+      SceGxmInternal:
+        kernel: false
+        nid: 0x3FE654E6
+        functions:
+          sceGxmCheckMemoryInternal: 0x88B424FB
+          sceGxmCreateRenderTargetInternal: 0xC8A0F04E
+          sceGxmGetDisplayQueueThreadIdInternal: 0xAB1D9466
+          sceGxmGetRenderTargetMemSizeInternal: 0x4DD98588
+          sceGxmGetTopContextInternal: 0x90EE6A28
+          sceGxmInitializedInternal: 0xC8E9F108
```
Comment (Rinnegatamante) [inline db.yml:2299]: I guess this and sceGxmIsInitialization are erroneously swapped?

Comment (Princess-of-Sleeping) [reply db.yml:2299]: One returns a bool value and the other returns 0 or an error code.

**Comment by d3m3vilurr** [comment]
@Rinnegatamante do you still have opinions?

File: db.yml [hunk 2]
```diff
@@ -2287,11 +2287,43 @@ modules:
           sceGxmVertexFence: 0xE05277D6
           sceGxmVertexProgramGetProgram: 0xBC52320E
           sceGxmWaitEvent: 0x8BD94593
+      SceGxmInternal:
+        kernel: false
+        nid: 0x3FE654E6
+        functions:
+          sceGxmCheckMemoryInternal: 0x88B424FB
```
Comment (d3m3vilurr) [inline db.yml:2294]: how about change name to `sceGxmInternal*` from `sceGxm*Internal`?

Comment (d3m3vilurr) [reply db.yml:2294]: ok we already have similar cases in db....

**Pull request merged**